src/db/couchPushEvents.ts

```diff
@@ -265,6 +276,27 @@ export async function* streamEvents(
   }
 }
 
+function fixJs(code: string): string {
+  return code.replace(/let|const/g, 'var')
```

Comment:
This looks like it may be dangerous because it will replace `let` and `const` inside of identifiers etc. Perhaps make the regex a little more succinct?

Reply:
Old version:
`'let letter = constant'.replace(/let|const/g, 'var')` === `'var varter = varant'`

New version:
`'let letter = constant'.replace(/\blet\b|\bconst\b/g, 'var')` === `'const letter = constant'`